app/Atro/Core/Templates/Repositories/Hierarchy.php

```diff
@@ -547,7 +547,7 @@ protected function afterRemove(Entity $entity, array $options = [])
 
     protected function afterRestore($entity, array $options = [])
     {
-        parent::afterRestore($entity, $options);
+        parent::afterRestore($entity);
```

Comment:
Why?

Reply:
after restore in the parent do not have the parameter $options